python/tests/engine/test_python.py

```diff
@@ -1102,7 +1105,7 @@ def test_convert_simple_pandas_type_uint8(self):
 
         # Act
         result = python_engine._convert_simple_pandas_dtype_to_offline_type(
-            dtype=np.dtype("uint8")
+            arrow_type=pa.from_numpy_dtype(np.dtype("uint8"))
```

Comment:
Not a big deal but instead of doing `pa.from_numpy_dtype(np.dtype("uint8"))` it would be cleaner to do `pa.uint8()` (this idea applies in multiple places in `test_python.py`)